docs/content/development/extensions-contrib/influx.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: doc_page
+---
+
+# InfluxDB Line Protocol Parser
+
+To use this extension, make sure to [include](../../operations/including-extensions.html) `druid-influx-extensions`.
+
+This extension enables Druid to parse [InfluxDB Line Protocol](https://docs.influxdata.com/influxdb/v1.5/write_protocols/line_protocol_tutorial/), a popular text-based timeseries metric serialization format. 
+
+## Line Protocol
+
+A typical line looks like this:
+
+```cpu,application=dbhost=prdb123,region=us-east-1 usage_idle=99.24,usage_user=0.55 1520722030000000000```
+
+which contains four parts:
+  - measurement: A string indicating what kind of measurement is represented (e.g. cpu, network, web_requests)
+  - tags: zero or more key-value pairs (i.e. dimensions)
+  - measurements: one or more key-value pairs; values can be numeric, boolean, or string
+  - timestamp: nanoseconds since Unix epoch (the parser truncates it to milliseconds)
+
+The parser extracts these fields into a map, giving the measurement the key `measurement` and the timestamp the key `_ts`. The tag and measurement keys are copied verbatum, so users should take care to avoid name collisions. It is up to the ingestion spec to decide which fields should be treated as dimensions and which should be treated as metrics (typically tags correspond to dimensions and values correspond to measurements).
```

Comment:
s/verbatum/verbatim/

Reply:
"metrics" and "measurements" seem to be used interchangeably here but I believe it should always be "metrics" no?

Reply:
Thanks for the feedback @josephglanville. Fixed first three as you suggested. `measurements` is the term used in the influx line protocol to refer to the second set of k/v pairs and I stuck with that in the docs here to avoid ambiguity with Druid `metrics`. That suggestion did lead me to fix this line "typically tags correspond to dimensions and values correspond to measurements" where I replaced "values" with "measurements".